core/tests/loadnext/src/sdk/ethereum/mod.rs

```diff
@@ -524,29 +566,43 @@ impl<S: EthereumSigner> EthereumProvider<S> {
             )
             .await?
         } else {
-            // TODO(EVM-571): This should be moved to the shared bridge, and the `requestL2Transaction` method
             let bridge_address =
-                bridge_address.unwrap_or(self.default_bridges.l1_erc20_default_bridge.unwrap());
+                bridge_address.unwrap_or(self.default_bridges.l1_shared_default_bridge.unwrap());
+            let bridgehub_address = self.bridgehub_address.unwrap();
+
+            let bridge_calldata = ethabi::encode(&(l1_token_address, amount, to).into_tokens());
+
             let contract_function = self
-                .l1_erc20_bridge_abi
-                .function("deposit")
-                .expect("failed to get function parameters");
+                .bridgehub_abi
+                .function("requestL2TransactionTwoBridges")
+                .expect("failed to get function");
+
             let params = (
-                to,
-                l1_token_address,
-                amount,
+                U256::from(self.l2_chain_id),
+                base_cost + operator_tip,
+                U256::zero(),
                 l2_gas_limit,
                 U256::from(L1_TO_L2_GAS_PER_PUBDATA),
+                Address::default(),
+                bridge_address,
+                U256::zero(),
```

Comment:
maybe worth adding a comment that we do not support depositing ETH to non-eth-based chains